package.json

```diff
@@ -43,6 +43,7 @@
     "chai": "^3.5.0",
     "enzyme": "^2.3.0",
     "jsdom": "^9.4.0",
+    "lodash": "4.14.1",
```

Comment:
I can add an `engines` directive to package.json that directs developers to use node 5+ when building, if you'd like.